bip-schnorr.mediawiki

```diff
@@ -123,7 +123,7 @@ All provided signatures are valid with overwhelming probability if and only if t
 ** Let ''c = r<sup>3</sup> + 7 mod p''.
 ** Let ''y = c<sup>(p+1)/4</sup> mod p''
 ** Fail if ''y<sup>2</sup> &ne; c''.
-** Let ''R<sub>i</sub> = (r, y)''.
+** Let ''R<sub>i</sub>'' be the point for which ''x(R<sub>i</sub>) = r'' and ''y(R<sub>i</sub>) = y''.
```

Comment:
Should the check on line above read `y^2 != c mod p`?

Reply:
Good catch. Opened PR to fix this https://github.com/sipa/bips/pull/20